### DISCUSSION TITLE
Payed application no longer working, asks for a subscription

### DISCUSSION THREAD

**Discussion by ctron** [blinksh/blink#1782] [Support] [answered]
So despite paying over 20 Euros for an application, it now is broken, no longer works and I should pay a monthly subscription for an SSH application.

It kind of feels like a scam. Especially as contacting the customer support doesn't give you any help either.

If you want to change to a subscription model for adding new features fine. But breaking the working application of people that payed is a different story.

So, how do I get back my money or get the working application back?

**Comment by carloscabanero** [comment, marked as answer]
Please read https://github.com/blinksh/blink/discussions/1778 and reach out to us

**Reply by ctron** [reply]
I do understand the frustration on the vendor side. However, paying a premium price, I also expect premium customer support.

So while I do believe the wording towards the customer could be improved, the issue indeed got resolved. Thanks.

**Reply by carloscabanero** [reply]
Yep, that is why we gave one year and a half to perform the migration and even tried to extend it performing manual migrations. Messaging can always be improved, we actually tried different messages during this year and a half, different ways to communicate it in the AppStore and inside the app. But the messages here + bad reviews + Reddit complaints are a 0.015% of all the users who migrated successfully. There is always room for improvement, but I think we did not do that bad.
